storage/aws/sqs/versions.tf

```diff
@@ -0,0 +1,5 @@
+terraform {
+  required_version = ">= 1.0"
+  required_providers {
+  }
```

Comment:
Shouldn't it be populated ?

Reply:
No, because I use nothing, I have nothing to populate it with. The only thing this module does is export some environment variables for ArmoniK.Core. If I populate it with "AWS" or "Kubernetes" the precommit hook will cry because none of them are being used.